src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -19,20 +19,15 @@ const styles = StyleSheet.create({
   },
   profile: {flexDirection: 'row', alignItems: 'center'},
   username: {
-    color: colors.black,
+    color: COLORS.black,
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 14
   },
   desc: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.gray
-  },
-  switch: {
-    fontFamily: fonts.inter[400],
-    fontSize: 12,
-    color: colors.gray
+    color: COLORS.gray8
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constants have been updated to use the new `COLORS` import. This is a good change as it maintains consistency across the codebase.
```diff
-    color: colors.black,
+    color: COLORS.black,

-    color: colors.gray
+    color: COLORS.gray8
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->